### PR TITLE
Fix semantics and rename http flag

### PR DESCRIPTION
**Pull request by kakkoyun**
* Fix semantics for http address flag
* Deprecate port flag
* Unify debuginfo flags

Signed-off-by: Kemal Akkoyun <kakkoyun@gmail.com>

### CODE REVIEW

File: pkg/parca/parca.go [hunk 1]
```diff
@@ -133,6 +135,11 @@ func Run(ctx context.Context, logger log.Logger, reg *prometheus.Registry, flags
 		defer closer()
 	}
 
+	if flags.Port != "" {
+		level.Warn(logger).Log("msg", "flag --port is deprecated, use --http-address instead")
```
Comment (brancz) [inline pkg/parca/parca.go:139]: We broke a bunch of flags in this release I think adding one more doesn’t make a difference

**Review by brancz — APPROVED** [review]
lgtm

**Review by yomete — APPROVED** [review]

File: pkg/parca/parca.go (hunk 1 above)
Comment (metalmatze) [reply pkg/parca/parca.go:139]: Agreed.

**Review by metalmatze — APPROVED** [review]

**Review by javierhonduco — APPROVED** [review]
🎉

File: pkg/parca/parca.go (hunk 1 above)
Comment (kakkoyun) [reply pkg/parca/parca.go:139]: 👍

Comment (kakkoyun) [reply pkg/parca/parca.go:139]: I'd done it but then reverted; it broke the snap. I think this is one of the oldest flags so let's don't break the backwards compatibility on this one.

**Pull request merged**